lib/output.ts

```diff
@@ -6,15 +8,17 @@ export function mismatchingVersionsToOutputLines(
 ): string[] {
   return mismatchingDependencyVersions.map(
     (obj) =>
-      `${obj.dependency} has more than one version: ${obj.versions
+      `${chalk.bold(
+        obj.dependency
+      )} has more than one version:\n  ${obj.versions
         .sort((a, b) => compareRanges(a.version, b.version))
         .map(
           (versionObj) =>
-            `${versionObj.version} (${versionObj.count} ${pluralizeUsage(
+            `${chalk.greenBright(versionObj.version)} (${versionObj.count} ${pluralizeUsage(
```

Comment:
Why green? Need a good reason to use colors.

Reply:
It makes it easier to read. I used a color that's similar to what we see in test output.

Reply:
Green indicates success. I don't think it makes sense here.

Reply:
Good point, I made it red.